fast/stages/02-security/variables.tf

```diff
@@ -174,6 +174,26 @@ variable "vpc_sc_perimeter_ingress_policies" {
   default = null
 }
 
+#TODO Ask Team
+variable "vpc_sc_dataplatform_projects" {
```

Comment:
These should not go here, as explained in the README the perimeters need to be controlled via the existing variables. Using variables for each subset of projects would make this unmanageable.